src/App.tsx

```diff
@@ -1,25 +1,44 @@
-import React from 'react';
+import React, { useState, useCallback } from 'react';
+import { Pagination } from './components/Pagination';
 import './App.css';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
```

Comment:
The `perPage` state is initialized with a magic number. It's a good practice to define such numbers as constants with descriptive names to improve readability and maintainability of the code.